resources/url_shortener/links_bulk.yml

```
@@ -0,0 +1,75 @@
+post:
+  operationId: link_bulk_create
+  summary: Bulk Create Links
+
+  description: >-
+    Given a bunch of long URLs, shortern all your URLs in a single request.
```

I'm thinking something like this might read a bit easier
```suggestion
    Shortens a list of links in a single request.
```